frontend/app/src/pages/people/PersonPage.tsx

```diff
@@ -82,7 +82,7 @@ export const PersonPage = observer(() => {
             height: '100%'
           }}
         >
-          <Panel isMobile={isMobile} style={{ paddingBottom: 0, paddingTop: 80 }}>
+          <Panel isMobile={isMobile} style={{ paddingBottom: isMobile ? 80 : 0, paddingTop: 80 }}>
```

Comment:
is there anyway to make this a styled component and use the isMobile in the styled component?